icu4c/source/common/unicode/platform.h

```diff
@@ -814,8 +814,8 @@ namespace std {
     /* Use the predefined value. */
 #elif defined(U_STATIC_IMPLEMENTATION)
 #   define U_EXPORT
-#elif defined(_MSC_VER) || (UPRV_HAS_DECLSPEC_ATTRIBUTE(dllexport) && \
-                            UPRV_HAS_DECLSPEC_ATTRIBUTE(dllimport))
+#elif defined(_MSC_VER) || (UPRV_HAS_DECLSPEC_ATTRIBUTE(__dllexport__) && \
```

Comment:
FYI/Note for others:
`UPRV_HAS_DECLSPEC_ATTRIBUTE` expands to `__has_declspec_attribute` (if defined), and `__has_declspec_attribute` is a clang language extension. Clang also supports using double-underscores for feature checking macros, so using `__dllexport__` should mean the same as using `dllexport`.

From https://clang.llvm.org/docs/LanguageExtensions.html#has-declspec-attribute :
> The attribute name can also be specified with a preceding and following `__` (double underscore) to avoid interference from a macro with the same name. For instance, `__dllexport__` can be used instead of `dllexport`.